### PR TITLE
Added initial DialogWindow template.

### PR DESCRIPTION
**Pull request by shaggygi**
This is a simple intro for a DialogWindow item template.  It could be improved later with additional common features and possibly MVVM support, but wanted to keep simple for now.

### CODE REVIEW

File: src/2019/ExtensibilityTemplatePack2019.csproj [hunk 1]
```diff
@@ -76,6 +76,9 @@
     <None Include="ProjectTemplates\VsixProjectWindow\VSCommandTable.cs" />
     <None Include="ProjectTemplates\VsixProjectCommand\VSCommandTable.cs" />
     <None Include="ProjectTemplates\VsixProject\VSCommandTable.cs" />
+    <Compile Include="ItemTemplates\DialogWindowCommunity\MyDialogWindow.xaml.cs">
```
Comment (madskristensen) [inline src/2019/ExtensibilityTemplatePack2019.csproj:79]: This file should not compile, right? It should be `<Content>`

File: src/2019/ExtensibilityTemplatePack2019.csproj [hunk 2]
```diff
@@ -211,6 +217,12 @@
       <HintPath>..\..\packages\TemplateBuilder.1.1.6.1\lib\TemplateBuilder.dll</HintPath>
     </Reference>
   </ItemGroup>
+  <ItemGroup>
```
Comment (madskristensen) [inline src/2019/ExtensibilityTemplatePack2019.csproj:220]: This should be `<content>`, right?

Comment (shaggygi) [reply src/2019/ExtensibilityTemplatePack2019.csproj:220]: Thanks for pointing out.  Good learning experience.

**Comment by madskristensen** [comment]
This is fantastic. Thank you!!

**Pull request merged**